docs/user/rstudio/ide/guide/ui/ui-panes.qmd

```diff
@@ -23,9 +23,9 @@ Additional source columns can be added or removed to the left of the 4 primary p
 
 The source pane allows users to view and edit various code-related files, such as .R, .rmd, .qmd, .py, .css, or general text files such as .txt or .md. By default it is the top-left panel and can be launched by opening any editable file in RStudio. Each additional file that is opened will be added as a new tab within the Source Pane.
 
-Additional source "columns" can be added from the **Global Options** > **Pane Layout** > **Add Source Column**, which provides the ability to work on two (or more) source files at a time within the Source Pane.
+Additional source "columns" can be added to the left of the four primary panes. To add additional source columns, from the RStudio menu: **Global Options** \> **Pane Layout** \> **Add Source Column**, which provides the ability to work on two (or more) source files at a time within the Source Pane.
 
-Individual source files can also be opened in their own window, rather than only within the **Source Pane**.  Source Windows allow you to edit files outside the main RStudio window. This is useful for splitting your work among multiple monitors, or devoting more space to your editor.
+Individual source files can also be opened in their own window, rather than only within the **Source Pane**. Source Windows allow you to edit files outside the main RStudio window. This is useful for splitting your work among multiple monitors, or devoting more space to your editor.
```

Comment:
Should "Source Windows" be capitalized?

Reply:
No, I think that's a description - changed to lower case.